### DISCUSSION TITLE
bootstrap server SAN with custom domain name

### DISCUSSION THREAD

**Discussion by kjbflood** [strimzi/strimzi-kafka-operator#8669] [General]
Hi, I'm trying to migrate our cluster config from using NGINX ingress to Traefik. As part of this change, I've modified the listeners to include a cluster-ip listener, which defines advertisedHost/Port values for the brokers with our internally-defined domain names. If I create a new certificate, I can define the server name as the bootstrap FQDN and include the broker FQDNs as SANs, but I'd like to continue to allow Strimzi to create the broker certs. 

When srimzi creates the certs, hostnames for the bootstrap server are included in the list of SANs, but none of them use our internal domain name. the include only .svc and .svc.cluster.local

When we used the ingress listener, the host property value was included in the broker certs as a SAN. Is there any method I can use do similar with a cluster-ip listener?

Thanks,
Kevin

**Comment by scholzj** [comment]
You can specify additional SANs in the `alternativeNames` section: https://strimzi.io/docs/operators/latest/full/configuring.html#property-listener-config-altnames-reference

**Reply by kjbflood** [reply]
Thanks Jakub, that worked perfectly.
Regards,
KEvin
